src/main/resources/migrations/V1__Initial.sql

```diff
@@ -1,12 +1,12 @@
 CREATE TABLE IF NOT EXISTS company
 (
-    id bigint GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
-    inn text NOT NULL UNIQUE,
-    ogrn text,
-    kpp text,
-    full_name_rus text,
+    id             bigint GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
nit: you shouldn't update previous migrations

Reply:
This happpened because I used "Reformat code".

Reply:
This happened because I used reformat code for all the packages. I was faced with a hash sum problem. Now I know it, I won't do it again for the whole project.